content/zh/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -51,7 +51,9 @@ which are configured in the API.
 
 <!--
 Admission controllers may be "validating", "mutating", or both. Mutating
-controllers may modify the objects they admit; validating controllers may not.
+controllers may modify related objects to the requests they admit; validating controllers may not.
+
+Admission controllers limit requests to create, delete, modify objects or connect to proxy. They do not limit requests to read objects.
```

Comment:
@mengjiao-liu 
I think I already synced with [30741](https://github.com/kubernetes/website/pull/30741).
What am I missing?

Reply:
Look at slack, I need to explain it to you.

Reply:
thanks for the explanation.